include/SampleTrack.h

```diff
@@ -58,6 +58,7 @@ class SampleTCO : public TrackContentObject
 		return m_sampleBuffer;
 	}
 
+	void setStartTimeOffset( MidiTime startTime );
```

Comment:
Are you sure this feature will not be useful for other TCOs? You do not have to implement those cases now.